src/main/io/displayport_msp_bf_compat.c

```diff
@@ -319,25 +319,21 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
 
     case SYM_AUTO_THR1:
         return BF_SYM_AUTO_THR1;
+        */
 
     case SYM_ROLL_LEFT:
-        return BF_SYM_ROLL_LEFT;
-
     case SYM_ROLL_LEVEL:
-        return BF_SYM_ROLL_LEVEL;
-
     case SYM_ROLL_RIGHT:
-        return BF_SYM_ROLL_RIGHT;
+        return BF_SYM_ROLL;
 
     case SYM_PITCH_UP:
-        return BF_SYM_PITCH_UP;
-
     case SYM_PITCH_DOWN:
-        return BF_SYM_PITCH_DOWN;
+        return BF_SYM_PITCH;
 
     case SYM_GFORCE:
-        return BF_SYM_GFORCE;
+        return 'G';
```

Comment:
I am on the fence on this one. @MrD-RC, @DzikuVx ?

Reply:
I’ve left my thoughts below. I don’t think it will surprise you much 😉

Reply:
I am obviously a bit more accommodating there. Using G for the icon is a simple enough change, and since it did not change the symbol for the other axii, I am leaving towards it being ok to change.